test/e2e/feature/feature.go

```diff
@@ -116,6 +132,10 @@ var (
 	//   is enabled such that passing CDI device IDs through CRI fields is supported
 	DRAAdminAccess = framework.WithFeature(framework.ValidFeatures.Add("DRAAdminAccess"))
 
+	// TODO: document the feature (owning SIG, when to use this feature for a test)
+	// OWNER: sig-node
+	// Testing downward API huge pages
+	DownwardAPIHugePages = framework.WithFeature(framework.ValidFeatures.Add("DownwardAPIHugePages"))
```

Comment:
Do we have extra requirements on test cluster? the DownwardAPIHugePages feature-gate was GA in 1.27 and remove in 1.29.

Reply:
So I don't think we should phase out feature tests if they require special configuration. I am not sure if default VMs use huge pages. So we would still want to keep features that require special setup.

I will not be doing that kind of cleanup in this PR at the moment.